tests/ci/cdk/cdk/codebuild/github_ci_linux_arm_omnibus.yaml

```diff
@@ -30,6 +30,16 @@ batch:
         compute-type: BUILD_GENERAL1_LARGE
         image: ECR_REPO_PLACEHOLDER:ubuntu-20.04_clang-7x_latest
 
+    - identifier: ubuntu2004_clang7x_aarch_fips
+      buildspec: ./tests/ci/codebuild/linux-aarch/run_fips_tests.yml
+      env:
+        type: ARM_CONTAINER
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_LARGE
+        image: ECR_REPO_PLACEHOLDER:ubuntu-20.04_clang-7x_latest
+        variables:
+          AWSLC_NO_ASM_FIPS: 1
```

Comment:
You probably answered this question before, Bryce; so this build will not use assembly files?

Reply:
The build includes multiple flavors.
When NO_ASM, ASAN is enabled.  
https://github.com/awslabs/aws-lc/blob/main/tests/ci/run_fips_tests.sh#L22